src/verify.v3.integration.test.ts

```diff
@@ -1,7 +1,574 @@
+      expect(obfuscated.issuer).toBe(undefined);
+      const fragments = await verifyRopsten(obfuscated);
+      const valid = isValid(fragments);
+      expect(fragments).toMatchInlineSnapshot(`
+        Array [
+          Object {
+            "data": true,
+            "name": "OpenAttestationHash",
+            "status": "VALID",
+            "type": "DOCUMENT_INTEGRITY",
+          },
+          Object {
+            "name": "OpenAttestationEthereumTokenRegistryStatus",
+            "reason": Object {
+              "code": 4,
+              "codeString": "SKIPPED",
+              "message": "Document issuers doesn't have \\"tokenRegistry\\" property or TOKEN_REGISTRY method",
+            },
+            "status": "SKIPPED",
+            "type": "DOCUMENT_STATUS",
+          },
+          Object {
+            "name": "OpenAttestationEthereumDocumentStoreStatus",
+            "reason": Object {
+              "code": 4,
+              "codeString": "SKIPPED",
+              "message": "Document issuers doesn't have \\"documentStore\\" or \\"certificateStore\\" property or DOCUMENT_STORE method",
+            },
+            "status": "SKIPPED",
+            "type": "DOCUMENT_STATUS",
+          },
+          Object {
+            "data": Object {
+              "details": Object {
+                "issuance": Object {
+                  "did": "did:ethr:0xE712878f6E8d5d4F9e87E10DA604F9cB564C9a89",
+                  "verified": true,
+                },
+              },
+              "issuedOnAll": true,
+              "revokedOnAny": false,
+            },
+            "name": "OpenAttestationDidSignedDocumentStatus",
+            "status": "VALID",
+            "type": "DOCUMENT_STATUS",
+          },
+          Object {
+            "name": "OpenAttestationDnsTxtIdentityProof",
+            "reason": Object {
+              "code": 2,
+              "codeString": "SKIPPED",
+              "message": "Document issuers doesn't have \\"documentStore\\" / \\"tokenRegistry\\" property or doesn't use DNS-TXT type",
+            },
+            "status": "SKIPPED",
+            "type": "ISSUER_IDENTITY",
+          },
+          Object {
+            "name": "OpenAttestationDnsDidIdentityProof",
+            "reason": Object {
+              "code": 0,
+              "codeString": "SKIPPED",
+              "message": "Document was not issued using DNS-DID",
+            },
+            "status": "SKIPPED",
+            "type": "ISSUER_IDENTITY",
+          },
+          Object {
+            "data": Object {
+              "did": "did:ethr:0xE712878f6E8d5d4F9e87E10DA604F9cB564C9a89",
+              "verified": true,
+            },
+            "name": "OpenAttestationDidIdentityProof",
+            "status": "VALID",
+            "type": "ISSUER_IDENTITY",
+          },
+        ]
+      `);
+      expect(valid).toBe(true);
+    });
+  });
+  describe("invalid documents", () => {
```

Comment:
for this failing test, maybe we could check the snapshot of the fragments that we expect to fail, only:
```
const fragment = fragments.find(ble => ble)
expect(fragment).toMatchInlineSnapshot()
```

something like this. (I think you decided to avoid to had to many snapshots)

Reply:
good idea, i'll add one: `findFailingFragment` later